### PR TITLE
add check to stop timbot from responding to himself

### PR DESCRIPTION
**Pull request by Daniel-Pivonka**

### CODE REVIEW

File: timbot.py [hunk 1]
```diff
@@ -51,61 +51,67 @@ def run_timbot():
 
 	# look for a message in the chat that starts with '@timbot .....'
 	if 'messages' in history:
-		for message in  history['messages']:
-			if 'text' in message:
-				message = message['text']
+		for data in  history['messages']:
+			if 'text' in data and 'user' in data:
```
Comment (timflannagan) [inline timbot.py:55]:
```suggestion
			if 'text' in data and 'user' in data and data['user'] != timbot_user_id_striped:
```

Comment (Daniel-Pivonka) [reply timbot.py:55]: That works too but i think its more readable separated. You are combined key checks and data checks in one statement.

Comment (nathan-weinberg) [reply timbot.py:55]: I agree with Dan; too many logic checks in a single line is a bad idea

**Review by nathan-weinberg — APPROVED** [review]
LGTM

**Pull request merged**